### DISCUSSION TITLE
Error: Cannot find module '.prisma/client/sql'

### DISCUSSION THREAD

**Discussion by pascalschaatsbaanrotterdam** [prisma/prisma#25381] [Q&A] [answered]
### Question

I am using Prisma TypedSQL ([https://www.prisma.io/docs/orm/prisma-client/using-raw-sql/typedsql](https://www.prisma.io/docs/orm/prisma-client/using-raw-sql/typedsql)) for a complex queries an everything run smoothly on my local environment. I just have an issue when I try to deploy the code with a dockerfile. Before it did work with `npx prisma generate` but when I added --sql I am getting a error. 

I am getting the following error: 

Error: Cannot find module '.prisma/client/sql'

```
# Stage 1: Build
FROM node:20-alpine AS build
WORKDIR /app

COPY package.json /app/package.json
COPY package-lock.json /app/package-lock.json
RUN npm ci

COPY prisma /app/prisma
COPY src /app/src
COPY *.json /app

RUN npx prisma generate --sql

RUN npm run build

# Stage 2: Run
FROM node:20-alpine
WORKDIR /app

COPY --from=build /app/dist ./dist
COPY package*.json ./
RUN npm ci --only=production

COPY --from=build /app/prisma ./prisma
COPY --from=build /app/node_modules/prisma ./node_modules/prisma

EXPOSE 3500
CMD ["sh", "-c", "npx prisma migrate deploy && node dist/src/index.js"]
```

Does anyone know what I am missing to make it work with --sql after npx prisma generate? 

I am using Prisma v5.20

**Comment by ludralph** [comment, marked as answer]
Hi @pascalschaatsbaanrotterdam 👋 

Can you add this line to your docker file and check if it resolves the issue for you?

```
COPY --from=build /app/node_modules/.prisma /app/node_modules/.prisma
```

**Reply by pascalschaatsbaanrotterdam** [reply]
Thanks! Solved the problem :)

**Reply by ludralph** [reply]
You're welcome 😄
